Formula/go.rb

```diff
@@ -3,9 +3,8 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
-    url "https://storage.googleapis.com/golang/go1.9.1.src.tar.gz"
-    mirror "https://fossies.org/linux/misc/go1.9.1.src.tar.gz"
```

Comment:
Why are we removing the mirror?

Reply:
They haven't updated it yet.

Reply:
It doesn't matter if they've updated it or not.

Reply:
I thought there was an audit for invalid mirrors now? Did I imagine that?

Reply:
It's --strict precisely due to this situation.

Reply:
Ah, yeah, I just run `--strict` on everything which explains why I hadn't really noticed the difference 😅.

Reply:
Ah, TIL about `--strict`! I'll use that next time 😄

Reply:
And voilà like clockwork the mirror is valid today. :tada: